packages/experimental/Checkbox/MIGRATION.md

```diff
@@ -0,0 +1,106 @@
+# Checkbox Migration
+
+## STATUS: WIP 🚧
+
+This Migration guide is a work in progress and is not yet ready for use.
+
+## Migration from v0 Checkbox
+
+### Component renames
+
+| v0 `Checkbox`                       | v1 `Checkbox`                       |
+| ----------------------------------- | ----------------------------------- |
+| `<Checkbox label="Hello, World" />` | `<Checkbox label="Hello, World" />` |
+
+### Props that remain as is
```

Comment:
NIT: "Props that remain unchanged" makes a bit more sense to me

Reply:
Thanks for the suggestions! Let me know if the new commit is better.